dask/array/lib/stride_tricks.py

```diff
@@ -0,0 +1,71 @@
+from ..overlap import rechunk_for_overlap, map_overlap, trim_chunks
+
+
+def _window_view(arr, window_shape, axis, fix_first_block, block_info):
+    """ core sliding window function"""
```

Comment:
```suggestion
    """Core sliding window function"""
```
Although it's probably worth either expanding the docstring or just deleting it - it doesn't add much value as is.